README.md

```diff
@@ -489,8 +499,28 @@ SIZE:int
 # Set the alpha of the dots (0=max opacity, 1=no opacity)
 ALPHA:float 
 
+# Whether to mix-up different groups' plotting order in UMAP; used when groups are plotted on top of each other.
+MIX_GROUPS:bool
+
 # Set the color mapping dictionary (name: {alias:alias, color:color})
 COLOR_MAPPINGS:Dict[str, Dict[str,str]]
+
+# Set the alias mapping key
+MAPPINGS_ALIAS_KEY:str
+# Set the color mapping key
+MAPPINGS_COLOR_KEY:str
+
+# Wether to show the baseline's ARI boxplot; used for marker ranking plots
+SHOW_BASELINE:bool
+
+# Define marker order for bubble plot
+ORDERED_MARKERS:List[str]
+
+# Define cell line order for bubble plot
+ORDERED_CELL_LINES:List[str]
+
+# Define a range for the y-axis break (used for marker ranking graph, if y-axis cut is needed)
+YAXIS_CUT_RANGES:Dict[str:Tuple[float,float], str:Tuple[float,float]]
```

Comment:
Sorry for commenting here again but basicaly I meant to put here (and in the relevant config) something like CELLPOSE config:
CELLPOSE = {
            'NUCLEUS_DIAMETER': int,
            'CELLPROB_THRESHOLD': int,
            'FLOW_THRESHOLD': float
        } 

meaning:
YAXIS_CUT_RANGES = {
   'UPPER_GRAPH': Tuple[float,float],
   'LOWER_GRAPH': Tuple[float,float]
}

since it shows the expected names for the keys.
In the current situation it's hard to know the relevant keys without going deep into the code